draft-patton-cfrg-vdaf.md

```diff
@@ -490,13 +494,18 @@ share or an error."}
   {{run-vdaf}}), and one of the input shares generated by the client
   (`input_share`). Its outputs is the Aggregator's initial preparation state.
 
-* `PrepState.next(inbound: Vec[Bytes]) -> outbound` is the deterministic
+* `PrepState.next(inbound: Optional[Bytes]) -> outbound` is the deterministic
   preparation-state update algorithm run by each Aggregator. It updates the
   Aggregator's preparation state (an instance of `PrepState`) and returns either
   its outbound message for the current round or, if this is the last round, its
   output share. An exception is raised if a valid output share could not be
-  recovered. The input of this algorithm is the sequence of inbound messages
-  from the previous round or, if this is the first round, an empty vector.
+  recovered. The input of this algorithm is the preprocessed inbound message or,
+  if this is the first round, `None`.
+
+* `prep_preprocess(agg_param, messages: Vec[Bytes]) -> preprocessed_message` is
+  a deterministic procedure used to preprocess the sequence of inbound messages
+  from the previous round into the input for the preparation-state update
+  algorithm.
```

Comment:
Can you elaborate a bit what the `prep_preprocess` step does and in what cases it is useful? This doesn't become clear from the description currently.

Reply:
I've tacked on a commit that will hopefully make this more clear.